src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -427,6 +437,21 @@ internal static unsafe int LastIndexOf(ReadOnlySpan<char> source, ReadOnlySpan<c
             }
         }
 
+        private static void InitCasingTable()
+        {
+            ushort []?[] table = new ushort []?[s_casingTableInit.Length];
```

Comment:
Is the whitespace between ushort and [] intended?

Reply:
The code below also has it, it can be removed.